### PR TITLE
Add all the fields

### PR DESCRIPTION
**Pull request by rachkor**
### Summary
Adds the temporal fields and all of the modified field as well as the one environmental field that was missing. Every field that has a button in the calculator now has a Dradis field that contains the button's value! 

> I assign all rights, including copyright, to any future Dradis work by myself to Security Roots.

### CODE REVIEW

File: app/assets/javascripts/dradis/plugins/calculators/cvss/calculator.js.coffee [hunk 1]
```diff
@@ -74,10 +74,38 @@
       issue_cvss += "#{output.baseIntegrity}\n\n"
       issue_cvss += "#[CVSSv3.BaseAvailability]#\n"
       issue_cvss += "#{output.baseAvailability}\n\n"
+      
+      issue_cvss += "#[CVSSv3.TemporalExploitCodeMaturity]#\n"
```
Comment (cbp1983) [inline app/assets/javascripts/dradis/plugins/calculators/cvss/calculator.js.coffee]:
```suggestion
      issue_cvss += "#[CVSSv3.TemporalExploitMaturity]#\n"
```

File: app/views/dradis/plugins/calculators/cvss/base/index.html.erb [hunk 2]
```diff
@@ -56,8 +56,24 @@ N/A
 #[CVSSv3.BaseConfidentiality]#
 #[CVSSv3.BaseIntegrity]#
 #[CVSSv3.BaseAvailability]#
+
+#[CVSSv3.TemporalExploitCodeMaturity]#
```
Comment (cbp1983) [inline app/views/dradis/plugins/calculators/cvss/base/index.html.erb]:
```suggestion
#[CVSSv3.TemporalExploitMaturity]#
```

File: app/views/dradis/plugins/calculators/cvss/issues/edit.html.erb [hunk 3]
```diff
@@ -63,8 +63,25 @@ N/A
 #[CVSSv3.BaseConfidentiality]#
 #[CVSSv3.BaseIntegrity]#
 #[CVSSv3.BaseAvailability]#
+
+#[CVSSv3.TemporalExploitCodeMaturity]#
```
Comment (cbp1983) [inline app/views/dradis/plugins/calculators/cvss/issues/edit.html.erb]:
```suggestion
#[CVSSv3.TemporalExploitMaturity]#
```

File: app/assets/javascripts/dradis/plugins/calculators/cvss/calculator.js.coffee [hunk 4]
```diff
@@ -74,10 +74,38 @@
...
+      issue_cvss += "#{output.exploitCodeMaturity}\n\n"
+      issue_cvss += "#[CVSSv3.TemporalRemediationLevel]#\n"
+      issue_cvss += "#{output.remediationLevel}\n\n"
+      issue_cvss += "#[CVSSv3.TemporalReportConfidence]#\n"
+      issue_cvss += "#{output.reportConfidence}\n\n"
```
Comment (etdsoft) [inline app/assets/javascripts/dradis/plugins/calculators/cvss/calculator.js.coffee]: why are you choosing to remove "temporal" from the variables? This breaks the convention we have elsewhere (e.g. see all the "environmental" entries in L85-L90)

Comment (rachkor) [reply app/assets/javascripts/dradis/plugins/calculators/cvss/calculator.js.coffee]: @etdsoft thanks for catching! Pushed the fix.

**Pull request merged**